java/mdsplus-api/src/main/java/mds/mdsip/MdsIp.java

```diff
@@ -75,7 +75,7 @@ public Message getMessage() throws MdsAbortException {
 		public final void run() {
 			try{
 				for(;;){
-					final Message new_message = Message.receive(MdsIp.this.connection, MdsIp.this.translisteners);
+					final Message new_message = Message.receive(MdsIp.this.connection, MdsIp.this.translisteners, -1);
```

Comment:
What does the -1 do?

Reply:
disables timeout for the receiver thread.